src/provable/curve-bigint.ts

```diff
@@ -9,20 +12,42 @@ import {
 } from './field-bigint.js';
 import { provable } from './provable-bigint.js';
 
-export { PublicKey, PrivateKey };
+export { Group, PublicKey, Scalar, PrivateKey, versionNumbers };
 
+// TODO generate
+const versionNumbers = {
+  field: 1,
+  scalar: 1,
+  publicKey: 1,
+  signature: 1,
+};
+
+type Group = { x: Field; y: Field };
 type PublicKey = { x: Field; isOdd: Bool };
 type Scalar = bigint;
 type PrivateKey = bigint;
 
-// TODO generate
-let FIELD_VERSION = 1;
-let PUBLIC_KEY_VERSION = 1;
+const Group = {
+  toProjective({ x, y }: Group): GroupProjective {
+    return Pallas.ofAffine({ x, y, infinity: false });
+  },
+  fromProjective(point: GroupProjective): Group {
+    let { x, y, infinity } = Pallas.toAffine(point);
+    if (infinity) throw Error('Group.fromProjective: point is infinity');
+    return { x, y };
+  },
+  get one(): Group {
```

Comment:
Suggest calling this `generator_mina` because when we update to the official Pasta curves it will mean there's another generator called `generator_pasta`